index.js

```diff
@@ -1,49 +1,21 @@
 import { parseToJSON } from "./src/parser.js";
-//import { program } from "commander";
-import axios from "axios";
-
-/**
- * Parses an OWL file to a json tree of nodes
- * @param {string} url The URL of the owl file in raw format e.g "https://raw.githubusercontent.com/edamontology/edamontology/main/releases/EDAM_1.25.owl"
- * @param {function} onSuccess The callback function to be executed after the tree is ready e.g (tree) => {console.log(tree)}
- * @param {function} onError The callback function to be executed in case of an error
- */
-const jsonTreeFromURL = (url, onSuccess, onError) => {
-  axios
-    .get(url)
-    .then((resp) => {
-      parseToJSON(resp.data, onSuccess);
-    })
-    .catch((err) => {
-      onError(err);
-    });
-};
+import { program } from "commander";
+import { writeJSONFile } from "./src/utils.js";
+import * as fs from "fs";
 
 /**
  * Parses an OWL file provided in the command line to a json tree of nodes. Outputs the tree to the console.
  * @param {string} filePath The path of the OWL file to be parsed
  */
 const jsonTreeFromFile = (filePath) => {
-  //to be implemented
+  var owlText = fs.readFileSync(filePath, "utf-8");
+  parseToJSON(owlText, writeJSONFile);
 };
 
-/**
- * Parses an OWL file provided in the command line to a list of RDFs. Outputs the list to the console.
- * @param {string} filePath The path of the OWL file to be parsed
- */
-const parseRDF = (filePath) => {};
-
-/*program.option(
-  "-jt,--jsontree [edam-owl-file]",
-  "Generate a json representation of the EDAM hierarchy"
-);
-
-//to be implemented
 program.option(
-  "-jld,--jsonld [edam-owl-file]",
-  "Generate a json-ld formatted version of EDAM"
+  "-jt,--jsontree [edam-owl-file]",
```

Comment:
I'd rather have the following syntax for the file:

```
edam2json jt [-o OUTPUT_FILE] EDAM_FILE_PATH 
```
-> the input OWL file path EDAM_FILE_PATH should be a mandatory argument
-> the output file should be printed to standard output, or written to OUTPUT_FILE if it has been specified

my explanation here might not be very understandable, please ask on gitter if it makes no sense :slightly_smiling_face:

Reply:
it works now like that -> edam2json -jt EDAM_dev.owl -o tree.json. Does this look good or should I reorder the output option?